indico/modules/logs/forms.py

```diff
@@ -0,0 +1,25 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2025 CERN
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see the
+# LICENSE file for more details.
+
+from wtforms.fields import StringField, TextAreaField
+from wtforms.validators import DataRequired, Optional
+
+from indico.util.i18n import _
+from indico.web.forms.base import IndicoForm
+from indico.web.forms.widgets import TinyMCEWidget
+
+
+class ResendEmailPrefaceForm(IndicoForm):
+    subject = StringField(_('Subject'), [DataRequired()])
+    preface = TextAreaField(_('Preface'), [Optional()],
```

Comment:
I don't think this is needed, it's only relevant when you have other validators that fail in case of no value.


```suggestion
    preface = TextAreaField(_('Preface'),
```